ext/system.py

```diff
@@ -0,0 +1,38 @@
+from datetime import datetime
+
+import disnake
+from disnake.ext import commands, tasks
+
+from utils.bot import Bot, Cog
+from utils.constants import GUILD_ID
+
+
+class SystemListeners(Cog):
+    @Cog.listener()
+    async def on_slash_command_error(self, inter: disnake.ApplicationCommandInteraction, err: commands.CommandError):
+        await inter.send(embed=disnake.Embed(color=0xFF0000, title="Error Occurred", description=f"```py\n{err}```"))
+        self.bot.log.error("Error occurred in command /%s", inter.application_command.name, exc_info=err)
+
+
+class SystemLoops(Cog):
+    def __init__(self, bot: Bot):
+        super().__init__(bot)
+        self.ban_checks.start()
```

Comment:
You can start or stop the tasks loop in cog_load and unload

Reply:
Doesn't really matter since we never unload